### ISSUE TITLE
[BUG] make_train data for 3D issue

### ISSUE THREAD

**Issue by s-udisha** [MouseLand/cellpose#1490]
## Before you fill out this form:
**Did you review the FAQ?**

- [ ] Yes, I reviewed the FAQ on the [cellpose ReadTheDocs](https://cellpose.readthedocs.io/en/latest/faq.html) 

**Did you look through previous (open AND closed) issues posted on GH?**
- [ ] Yes, I [searched for related previous GH issues](https://github.com/MouseLand/cellpose/issues?q=is%3Aissue).

### **Describe the bug**
I was trying to train a model on 3D data and to make the data for training, I used make_train.py which requires segmentation masks to be in .npy format. This format expect that ground truth is generated in cellpose GUI but if ground truth exist already in other format it's not accounted into the code.
As I am using simulated data, the ground truth is in tiff format so to try it out I converted one of the image to .npy and rewrap it into pickle dictionary. The issue with make_train.py is that mask crop saving is within an error loop and after fixing it, mask slices are saved and I wanted the crop size = 256 not 512 because later during training 256 is the size of images going as input. As I had image size of less than 256 in one axis the problem is there is no padding happening in make train to account for such cases and also dropping of slices based on minimum number of masks present is not within make_train file but happening during training, due to which after drop of number of slices training data becomes less than expected.
Another problem is with lattice light sheet dataset after deskewing data doesn't look cuboidical but parallelogram shape so these slices in YZ and ZX include sharp wedge where background is including. Is this a problem to include in training data? 

### **Run log**
Please post all command line/notebook output for us to understand the problem. For this please make sure you are running with verbose mode on. So command line, with `--verbose` tag, or in a notebook first run 
```from cellpose import io 
logger = io.logger_setup()
``` 
before running any Cellpose functions.

<details><summary>Run logs</summary>
your logs here.
</details>
